src/index.html

```diff
@@ -1,12 +1,288 @@
+          </div>
+
+          <a class="button" data-qa="card-hover">BUY</a>
+        </div>
+
+        <div class="product">
+          <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="product__image"
+          >
+          <h2 class="product__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="product__code">Product code: 195434</p>
+
+          <div class="stars-review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="stars-review__content">Reviews: 5</p>
+          </div>
+
+          <div class="price">
+            <p class="price__text">Price:</p>
+            <p class="price__value">$2,199</p>
+          </div>
+
+          <a class="button">BUY</a>
+        </div>
+
+        <div class="product">
+          <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="product__image"
+          >
+          <h2 class="product__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="product__code">Product code: 195434</p>
+
+          <div class="stars-review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="stars-review__content">Reviews: 5</p>
+          </div>
+
+          <div class="price">
+            <p class="price__text">Price:</p>
+            <p class="price__value">$2,199</p>
+          </div>
+
+          <a class="button">BUY</a>
+        </div>
+
+        <div class="product">
+          <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="product__image"
+          >
+          <h2 class="product__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="product__code">Product code: 195434</p>
+
+          <div class="stars-review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="stars-review__content">Reviews: 5</p>
+          </div>
+
+          <div class="price">
+            <p class="price__text">Price:</p>
+            <p class="price__value">$2,199</p>
+          </div>
+
+          <a class="button">BUY</a>
+        </div>
+      </div>
+      <div class="container">
```

Comment:
do you really need the second flex container?
if there will be more than 4 cards?
![image](https://user-images.githubusercontent.com/87998640/221652891-b76db0ce-038e-44a6-9a49-48e9b3a45c27.png)